src/concrete/ml/quantization/quantized_module.py

```diff
@@ -461,7 +469,7 @@ def _fhe_forward(self, *q_x: numpy.ndarray, simulate: bool = True) -> numpy.ndar
                 Default to True.
 
         Returns:
-            (numpy.ndarray): Predictions of the quantized model, with integer values.
+            (Tuple[numpy.ndarray, ...]): Predictions of the quantized model, with integer values.
```

Comment:
technically it should be `Union[numpy.ndarray, Tuple[numpy.ndarray, ...]]`

Reply:
indeed